cli/cmd/root.go

```diff
@@ -20,15 +23,16 @@ const (
 
 var appSlugOrID string
 var apiToken string
-var apiOrigin = "https://api.replicated.com/vendor"
+var platformOrigin = "https://api.replicated.com/vendor"
+var shipOrigin = "https://g.replicated.com/graphql"
 
 func init() {
 	RootCmd.PersistentFlags().StringVar(&appSlugOrID, "app", "", "The app slug or app id to use in all calls")
 	RootCmd.PersistentFlags().StringVar(&apiToken, "token", "", "The API token to use to access your app in the Vendor API")
 
 	originFromEnv := os.Getenv("REPLICATED_API_ORIGIN")
 	if originFromEnv != "" {
-		apiOrigin = originFromEnv
+		platformOrigin = originFromEnv
```

Comment:
do you wanna add something similar to here for ship? so we can test with staging?